agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -1569,8 +1586,9 @@ void ap_manager_thread::handle_hostapd_attached()
     LOG(INFO) << " vht_center_frequency = " << ap_wlan_hal->get_radio_info().vht_center_freq;
     LOG(INFO) << " current bw = " << ap_wlan_hal->get_radio_info().bandwidth;
     LOG(INFO) << " supported_channels = " << std::endl
-              << get_radio_supported_channels_string(ap_wlan_hal);
-
+              << get_radio_supported_channels_string(notification->params().supported_channels);
+    LOG(INFO) << " hw_supported_channels = " << std::endl
```

Comment:
The terminology should be as follows:
supported channels - these are the supported channels by the radio we read from NL
preferred channels - these are the preferred channels we get from the ACS report

Reply:
Changed